content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/saml-sso.md

```diff
@@ -45,6 +39,141 @@ We don’t synchronize user datastores, so make sure that all users:
 
 If you have problems setting up SAML SSO, see our troubleshooting tips below.
 
+## Configuration Instructions for Specific Identity Providers
+
+<!--You can configure SAML SSO with your preferred identity provider, beyond those listed in this section.
+
+Here are configuration instructions for some popular identity providers:-->
+
+We support the following identity providers for SAML SSO:
+
+- [Azure AD](#azure-ad)
+- [Duo](#duo)
+- [Google](#google)
+- [Okta](#okta)
+- [OneLogin](#onelogin)
+
+### Azure AD
+
+To configure SAML SSO with Azure Active Directory (Azure AD):
+
+1. In Azure AD, create an [enterprise non-gallery application](https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal) for Cobalt.
+1. [Enable SSO](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal-setup-sso) for the application. If available, follow the instructions that you see in the UI.
+    - Verify that the single sign-on method for your application is SAML.
+    - Under **Basic SAML Configuration**, enter:
+       - **Identifier** (Entity ID): `https://api.cobalt.io/users/saml/metadata`
+       - **Reply URL** (Assertion Consumer Service URL): `https://api.cobalt.io/users/saml/auth`
+       - **Sign on URL**: Leave this field blank for IdP-initiated SAML.
+       - **Relay State**: Leave this field blank.
+       - **Logout URL**: leave this field blank.
+    - Under **User Attributes & Claims**, add custom attribute mappings to your SAML token attributes configuration.
+       - `givenname`: `user.givenname`
+       - `surname`: `user.surname`
+       - `emailaddress`: `user.mail`
+       - `name`: `user.userprincipalname`
+       - `Unique User Identifier`: `user.userprincipalname`
```

Comment:
should unique user identifier be lower case?

Reply:
i looked in Azure Ad and yes, they do have "Unique User Identifier" as title case